docs/pages/example/custom-style-id.html

```diff
@@ -2,7 +2,7 @@
 <script>
     var map = new mapboxgl.Map({
         container: 'map', // container id
-        style: 'mapbox://styles/mapbox/dark-v10', //hosted style id
+        style: 'mapbox://styles/examples/cke97f49z5rlg19l310b7uu7j', // custom style url id
```

Comment:
Maybe add where to find the style url?

```suggestion
        style: 'mapbox://styles/examples/cke97f49z5rlg19l310b7uu7j', // custom style url from https://studio.mapbox.com/
```

Reply:
This map. OMG. amazing.